app/views/layouts/_languages.html.erb

```diff
@@ -130,3 +130,38 @@
   </dl>
   <a class="close-reveal-modal">&#215;</a>
 </div>
+
+<script type="text/javascript">
```

Comment:
can we move all this js out of the page itself? Since is a general thing, shown in every page, you can put it into airesis.coffee.